javav2/example_code/route53recoverycluster/src/main/resources/routingControlStates.json

```diff
@@ -0,0 +1,8 @@
+{
+  "RoutingControlStates": [
+    {
+      "RoutingControlArn": "arn:aws:route53-recovery-control::308833258515:controlpanel/930a847cd870495da38a6da151c61aca/routingcontrol/1599063100e64982",
```

Comment:
This should use example numbers for AWS account, etc., to make it clear they are for example only (or don't include this in the example).

Reply:
Updated to example arn